index.html

```diff
@@ -611,6 +677,12 @@ <h2>Process policy headers</h2>
           </p>
 
           <dl>
+            <dt><a>received IP address</a></dt>
+            <dd>
+            the IP address of the <a>server</a> that the user agent received
+            <var>response</var> from

```

Comment:
I see that [response](https://w3c.github.io/network-error-logging/#process-policy-headers) is defined by reference to [RFC7230](https://tools.ietf.org/html/rfc7230#section-2.1) instead of [Fetch](https://fetch.spec.whatwg.org/#responses). That's probably the wrong choice, particularly because you can pretty easily extend Fetch's data type if you need extra data stored in it.

In this case, I think you need to add the server IP address to https://fetch.spec.whatwg.org/#concept-connection and a way to get the connection for a https://fetch.spec.whatwg.org/#concept-response. @annevk, does that sound right?

Reply:
I used the lower-level references so that it's hopefully more clear how a non-browser user agent would use NEL — for instance, a native app using OkHttp to make API requests.  That's also the rationale behind the particular wording I chose when I introduced phases earlier on.  I see what you're saying though — jumping through hoops to make this more generic might not be worth the trouble.

Reply:
Ditto — I've updated the reference so that it points to Fetch intstead of RFC 7230, but otherwise I've just added an editor's note that we want to plumb this through more explicitly in Fetch.

Reply:
Ack